sympy/physics/control/lti.py

```diff
@@ -422,6 +423,89 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
     return wrapper
 
+def routh_hurwitz_table(system, first_col=False ,reduce=False):
+    """
+    Creates the Routh-Hurwitz table for the given transfer function.
+
+    Parameters
+    ==========
+    system : TransferFunction
+        The transfer function for which the Routh-Hurwitz table is to be generated.
+
+    first_col : bool, optional
+        If True, returns only the first column of the Routh-Hurwitz table.
+        Default is False.
+
+    reduce : bool, optional
+        If True, simplifies the entries in the Routh-Hurwitz table.
+        Default is False.
+
+    Returns
+    =======
+    table : Matrix
+        The Routh-Hurwitz table as a SymPy Matrix.
+
+    first_column : list, optional
+        If `first_col` is True, returns the first column of the table as a list.
```

Comment:
I think that this can be unnecessary option for users. It is easy to get the first column of a matrix like `table[:, 0]`